da/da_test.go

```
@@ -41,346 +31,244 @@ const (
 	}
 
 	for batch := uint64(0); batch < numBatches; batch++ {
-		headers := make([]*types.SignedHeader, numHeaders)
+		headers := make([]coreda.Blob, numHeaders)
 		for i := range headers {
-			headers[i], _ = types.GetRandomBlock(batch*numBatches+uint64(i), rand.Int()%20, "doTestSubmitRetrieve") //nolint:gosec
+			headers[i] = make([]byte, 1234)
 		}
 		submitAndRecordHeaders(headers)
-		time.Sleep(time.Duration(rand.Int63() % MockDABlockTime.Milliseconds())) //nolint:gosec
+		time.Sleep(time.Duration(rand.Int63() % MockDABlockTime.Milliseconds()))
```

golangci-lint (1.64.8)

[high] G404: Use of weak random number generator (math/rand or math/rand/v2 instead of crypto/rand)

(gosec)